src/components/CollectionLink/CollectionLink.tsx

```diff
@@ -32,22 +26,18 @@ export const CollectionLink = ({
             alt={background.alt}
             className={styles.bgImage}
           />
-          <span className={styles.overlay}></span>
         </div>
       )}
-      <div className={styles.label}>{label}</div>
+      <div className={styles.labelContainer}>
+        <h2 className={styles.label}>{label}</h2>
```

Comment:
No need for heading tag. Could cause heading hierarchy issues depending on where these will be placed.